src/index/base.cpp

```diff
@@ -23,7 +23,7 @@ static void FatalError(const char* fmt, const Args&... args)
     SetMiscWarning(strMessage);
     LogPrintf("*** %s\n", strMessage);
     uiInterface.ThreadSafeMessageBox(
-        "Error: A fatal internal error occurred, see debug.log for details",
```

Comment:
Aren't we intentionally keeping the message vague here to avoid confusing users?